evm/src/main/java/org/hyperledger/besu/evm/gascalculator/stateless/Eip4762AccessWitness.java

```diff
@@ -288,6 +288,7 @@ public long touchAddressAndChargeGas(
       final int accessMode) {
     final short accessEvents = touchAddress(address, treeIndex, subIndex, accessMode);
     long gas = 0;
+    LOG.atDebug().log(">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>");
```

Comment:
I think we can remove it

Reply:
To me it is useful for debugging because it tells me what has been charged in a single touch call, because we could have multiple charges in one call.
Maybe we can do a pretty print at the end to show this information at once instead. I'll have a look